/controller:
    ros__parameters:
        radius: 25.0
        cy: 70.0
        wheel_base: 1.88
        max_steer_angle: 0.3
        max_steer_rate: 0.3
        max_velocity: 10.0
        max_acc: 0.8
        traj_dl: 0.2
        traj_length: 50.0
        timer_period: 0.05
        mpc_steps: 6
        mpc_dt: 0.15
        kcte: 1.1
        kepsi: 2.0
        kev: 0.1
        ksteer_cost: 5.0